### PR TITLE
Add a list of talks to "schedule" page

### PR DESCRIPTION
**Pull request by yarikoptic**
@christian-monch , @mih, @candleindark please reconfirm the intent and perspective titles 

@jsheunis @mslw - planing to present any?

### CODE REVIEW

**Comment by yarikoptic** [comment]
@jsheunis if you see how to easily make it prettier -- feel welcome to push tune up.

**Comment by mslw** [comment]
I think the simple list looks fine. I tried italicizing titles to see how that would work, but I think that's not needed.
> planing to present any?

I added my proposed topics to the cryptpad document, but IIRC the consensus after seeing the submissions was that we don't need more than a few contributions from team members, so I wouldn't list mine (and treat them as reserve in case of schedule changes).

File: content/schedule.md [hunk 1]
```diff
@@ -20,3 +20,24 @@ weight: 4
...
+9. OpenNeuro and DataLad (Hardcastle, Nell; Stanford University)
+10. fMRI Pipelines on HPC with DataLad and ReproMan (Wexler, Joe; Stanford University)
+11. Workflow provenance-based scheduling (Martinez, Pedro; University of Calgary)
+12. Reproducible and replicable data science in the presence of patient  confidentiality concerns by utilizing git-annex and the Data Science Orchestrator (Brechtel, Markus Katharina; Research Group Cohorts in Infectious Disease and Cancer Research, University Hospitals Cologne and Frankfurt, German Centre for Infectious Diseases Research AND Kaluza, Philipp; Ghostroute Consulting)
+13. Neuroscientific data management using DataLad (Kosciessa, Julian; Donders Institute for Brain, Cognition and Behaviour, Radboud University)
+14. datalad-annex: a git-remote helper to deposit a repository in any place reachable by a git-annex special remote implementation (Hanke, Michael; Forschungszentrum Jülich)
+15. datalad-registry: a registry of datalad datasets with metadata extracted (To, Isaac; Dartmouth College, Center for Open Neuroscience)
```
Comment (candleindark) [inline content/schedule.md:41]: What about "datalad-registry: a registry of datalad datasets with extracted metadata" or simply "datalad-registry: a registry of datalad datasets with metadata"

Comment (yarikoptic) [reply content/schedule.md:41]: make it as a github suggestion: use 
image

Comment (candleindark) [inline content/schedule.md:41]:
```suggestion
15. DataLad-Registry: A Registry of DataLad Datasets with Metadata (To, Isaac; Dartmouth College, Center for Open Neuroscience)
```

**Comment by yarikoptic** [comment]
> I think the simple list looks fine.

ok then, let's proceed!

**Pull request merged**